<ui version="4.0" >
 <class>XUPProjectManager</class>
 <widget class="pDockWidget" name="XUPProjectManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>573</height>
   </rect>
  </property>
  <property name="allowedAreas" >
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle" >
   <string>Project Manager</string>
  </property>
  <widget class="QWidget" name="wCentral" >
   <layout class="QVBoxLayout" name="verticalLayout_3" >
    <item>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lProjects" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Projects:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbProjects" />
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QToolBar" name="tbActions" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QPushButton" name="pbFiltered" >
        <property name="text" >
         <string>Filtered</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbNative" >
        <property name="text" >
         <string>Native</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="swPages" >
      <widget class="QWidget" name="wFiltered" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QTreeView" name="tvFiltered" >
          <property name="uniformRowHeights" >
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus" >
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wNative" >
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QTreeView" name="tvNative" >
          <property name="uniformRowHeights" >
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus" >
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteLog" >
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="aDebug" >
   <property name="icon" >
    <iconset resource="../../resources/resources.qrc" >
     <normaloff>:/temp/icons/temp/kcmsystem.png</normaloff>:/temp/icons/temp/kcmsystem.png</iconset>
   </property>
   <property name="text" >
    <string>Debug</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global" >pDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
